src/lib.rs

```diff
@@ -16,6 +16,20 @@
 //!
 //! ```no_run
 //! let mut roots = rustls::RootCertStore::empty();
+//! let root_certs = &rustls_native_certs::load_native_certs()
+//!     .expect("could not load platform certs")
+//!     .iter()
+//!     .map(|c| c.0.clone())
```

Comment:
This `clone` is my clumsy way to avoid moving the `Vec<u8>`. I think probably we want to take a ref, but I wanted to put this draft up for feedback before logging off.

Reply:
Ah, well `add_parseable_certificates` takes `Vec<u8>` so having `Vec<&u8>` won't be helpful. The clone might be best after all.